packages/esm-cervical-cancer-app/package.json

```diff
@@ -0,0 +1,142 @@
+  "main": "src/index.ts",
+  "license": "MIT",
+  "homepage": "https://github.com/UCSF-IGHS/openmrs-esm-ohri#readme",
+  "scripts": {
+    "start": "openmrs develop",
+    "serve": "webpack serve --mode=development",
+    "debug": "npm run serve",
+    "build": "webpack --mode production",
+    "analyze": "webpack --mode=production --env.analyze=true",
+    "lint": "eslint src --ext tsx",
+    "prettier": "prettier --config prettier.config.js --write \"src/**/*.{ts,tsx}\"",
+    "typescript": "tsc",
+    "prepublishOnly": "npm run build",
+    "test": "jest --config ./jest.config.js --passWithNoTests",
+    "coverage": "npm test -- --coverage",
+    "badges": "npm run coverage && jest-coverage-badges --output './badges' "
+  },
+  "browserslist": [
+    "extends browserslist-config-openmrs"
+  ],
+  "keywords": [
+    "openmrs"
+  ],
+  "publishConfig": {
+    "access": "public"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "pretty-quick --staged && concurrently 'npm:test' 'npm:typescript' 'npm:lint'"
+    }
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/UCSF-IGHS/openmrs-esm-ohri.git"
+  },
+  "bugs": {
+    "url": "https://github.com/UCSF-IGHS/openmrs-esm-ohri/issues"
+  },
+  "dependencies": {
+    "@carbon/icons-react": "^10.49.0",
+    "ace-builds": "^1.4.12",
+    "carbon-components-react": "^7.56.0",
+    "enzyme": "^3.11.0",
+    "enzyme-adapter-react-16": "^1.15.6",
+    "formik": "^2.2.6",
+    "jest-coverage-badges": "^1.0.0",
+    "lodash-es": "^4.17.15",
+    "moment": "^2.29.1",
+    "react-ace": "^9.4.4",
+    "react-anchor-link-smooth-scroll": "^1.0.12",
+    "react-markdown": "^7.1.0",
+    "react-scroll": "^1.8.2",
+    "react-test-renderer": "^16.9.0",
+    "react-waypoint": "^10.1.0",
+    "semver": "^7.3.5",
+    "systemjs-webpack-interop": "^2.3.2",
+    "yup": "^0.29.1"
+  },
+  "peerDependencies": {
+    "@openmrs/esm-framework": "3.x",
+    "@openmrs/esm-patient-common-lib": "3.x",
+    "carbon-components": "^10.19.0",
+    "carbon-icons": "^7.0.7",
+    "dayjs": "^1.8.16",
+    "react": "^16.9.0",
+    "react-i18next": "^11.3.4",
+    "swr": "^1.1.2"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.5.5",
+    "@babel/plugin-proposal-class-properties": "^7.5.5",
+    "@babel/preset-env": "^7.5.5",
+    "@babel/preset-react": "^7.0.0",
+    "@babel/preset-typescript": "^7.3.3",
+    "@babel/runtime": "^7.17.8",
+    "@openmrs/esm-framework": "^3.2.1-pre.1055",
+    "@openmrs/esm-patient-common-lib": "^3.1.1-pre.1346",
+    "@testing-library/dom": "^7.20.0",
+    "@testing-library/jest-dom": "^5.16.3",
+    "@testing-library/react": "^10.4.3",
+    "@testing-library/user-event": "^12.0.11",
+    "@types/carbon-components-react": "^7.36.1",
+    "@types/jest": "^24.9.1",
+    "@types/lodash": "^4.14.161",
+    "@types/react": "^16.9.2",
+    "@types/react-dom": "^16.9.0",
+    "@types/react-router": "^5.1.7",
+    "@types/react-router-dom": "^5.1.5",
```

Comment:
take a look at the dev dependencies in the other esm...u might not need most of these since they're already in the main package.json

Reply:
Alright, let me take a look